adafruit_bitbangio.py

```diff
@@ -114,14 +122,14 @@ def scan(self):
                     found.append(address)
         return found
 
-    def writeto(self, address, buffer, *, start=0, end=None):
+    def writeto(self, address: int, buffer: ReadableBuffer, *, start=0, end=None) -> None:
```

Comment:
Looks like it just got missed here, but `start` is `int`, and `end` is `Optional[int]`!